lms/envs/aws.py

```diff
@@ -974,6 +974,13 @@
     ENTERPRISE_SUPPORT_URL
 )
 
+# A shared secret to be used for encrypting passwords passed from the enterprise api
+# to the enteprise reporting script.
+ENTERPRISE_REPORTING_SECRET = ENV_TOKENS.get(
+    'ENTERPRISE_REPORTING_SECRET',
+    ENTERPRISE_REPORTING_SECRET
+)
```

Comment:
Will the value of `ENTERPRISE_REPORTING_SECRET` set in the business-secure repo get picked up here? Or do we need to set the value somewhere else as well? @fredsmith

Reply:
I have the same question. I almost opened a PR in edx-internal but then it said something about no secrets, so I am not sure where to put it... I don't believe business-secure would get pulled in by default since the stuff I have there is the first stuff to exist.

Reply:
Also, is it necessary to block merging of this PR on having that final value defined in the right place? It's use is limited to the model I introduced, which is useless without the rest of everything.

Reply:
This will probably need to go in the edx-secret repo. Note that it'll need an edx/configuration change as well so that the variable propagates into the right config file. I don't think we need to block on that though.